src/autoscaler/metricscollector/exampleconfig/example.yml

```diff
@@ -12,5 +15,4 @@ db:
   metrics_db_url: "postgres://postgres:postgres@localhost/autoscaler?sslmode=disable"
 collector:
   refresh_interval_in_seconds: 60
```

Comment:
If you specify the type as a time.Duration, it can support 30s or 30m, really any of the time units.